internal/experimental/experimental.go

```diff
@@ -0,0 +1,77 @@
+package experimental
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	envName = "NOTATION_EXPERIMENTAL"
+	enabled = "1"
+)
+
+// IsDisabled determines whether an experimental feature is disabled.
+func IsDisabled() bool {
+	return os.Getenv(envName) != enabled
+}
+
+// Error returns an error for a disabled experimental feature.
+func Error(description string) error {
+	return fmt.Errorf("%q is marked as experimental and not enabled by default. To use it, please set %s=%s in your environment", description, envName, enabled)
```

Comment:
Had a try on the command on Linux, got the following error message:
`Error: "using flags \"--flag-a,--flag-b\" in \"notation sign\"" is marked as experimental and not enabled by default. To use it, please set NOTATION_EXPERIMENTAL=1 in your environment`
The above error message is not very easy to read through. How about:
```suggestion
	return fmt.Errorf("%s been marked as experimental and not enabled by default. To use it, please set %s=%s in your environment", description, envName, enabled)
```